### ISSUE TITLE
font and alignment issues

### ISSUE THREAD

**Issue by prasad471** [marcbachmann/node-html-pdf#476]
PDF that is generated in the local machine is fine. But when code deployed to a linus server the generated pdf font size is enlarged and also alignment changed.

**Comment by agiratech-reddysai** [comment]
Having the same issue, did you find any solution?

**Comment by prasad471** [comment]
No

**Comment by KrifaYounes** [comment]
same problem someone have an idea ?
letter spacing on linux server

**Comment by KrifaYounes** [comment]
It's possible to overwrite this file in a lambda function ?

**Comment by prasad471** [comment]
Hi @agiratech-reddysai Thanks for the solution, is there any effect on the other views of the application if we overwrite the /etc/fonts/conf.d/51-local.conf file.

**Comment by agiratech-reddysai** [comment]
@prasad471, it won't make any issues. If you have any doubt, create a new(unique) conf file and include that file inside `51-local.conf`.

**Comment by agiratech-reddysai** [comment]
@KrifaYounes , check this https://stackoverflow.com/questions/46486261/include-custom-fonts-in-aws-lambda

**Comment by rohitkhadatkar** [comment]
> @prasad471, it won't make any issues. If you have any doubt, create a new(unique) conf file and include that file inside `51-local.conf`.

It won't allow overriding `51-local.conf` file. we create a new file but didn't get succeed. can you suggest any other solution? PDF generating in local showing correct font family but not on the ubuntu server.

**Comment by intermx-reddysai** [comment]
@rohitkhadatkar , what was the issue to do override? if it is a permission issue, change it try once. Or, did you try to edit any other file inside `conf.d` directory?

**Comment by rohitkhadatkar** [comment]
<p style="margin: 0in 0in 0.0001pt; text-align: justify; line-height: 115%; font-size: 11pt; font-family: Calibri, sans-serif;"><span style="color: #595959;">This is testing content</span></p>

**Comment by rohitkhadatkar** [comment]
`<p style="margin: 0in 0in 0.0001pt; text-align: justify; line-height: 115%; font-size: 11pt; font-family: Calibri, sans-serif;"><span style="color: #595959;">This is test content</span></p>`

This works fine when we did in local, but not on the ubuntu server didn't get the correct font family. 
We change file permission but override the above changes successfully.

**Comment by intermx-reddysai** [comment]
The font is may not be available on the Server where you are running the app. Copy the font files into the `/usr/share/fonts/` and `/usr/local/share/fonts/` directories.

**Comment by rohitkhadatkar** [comment]
We have installed mscorefont in the ubuntu server. do we need to make some settings to utilize this installed font? 
As you said earlier copy the font file. from where should I download the font file and copy in /usr/share/fonts/ and /usr/local/share/fonts/ directories.

We have this type of folder font structure: 
/usr/share/fonts/truetype

**Comment by agiratech-reddysai** [comment]
Yes, copy the installed font into the /usr/share/fonts/ directory and do the same in the  /usr/local/share/fonts/ directory. I have solved the font-family related issue like this way. I hope, this works you too.

**Comment by rohitkhadatkar** [comment]
> Yes, copy the installed font into the /usr/share/fonts/ directory and do the same in the /usr/local/share/fonts/ directory. I have solved the font-family related issue like this way. I hope, this works you too.

We tried with your suggested method but didn't get success. 
Here is the our system path:

**Comment by rohitkhadatkar** [comment]
> Yes, copy the installed font into the /usr/share/fonts/ directory and do the same in the /usr/local/share/fonts/ directory. I have solved the font-family related issue like this way. I hope, this works you too.

We tried with your suggested method but didn't get success.
Here is our system path:
 /usr/share/fonts/truetype/newfonts and /usr/local/share/fonts/truetype/newfonts

We keep our all newly installed TTF file inside a new fonts folder then used 'fc-cache -f -v' command.

**Comment by agiratech-reddysai** [comment]
Could you please move the newfonts directory from truetype to fonts? like `/usr/local/share/fonts/newfonts` and `/usr/share/fonts/newfonts`. Revert me if it does not work, I will look at it with this font.

**Comment by rohitkhadatkar** [comment]
We will do and let you know if we need anything. Thanks for helping.

**Comment by mahendra119** [comment]
> Could you please move the newfonts directory from truetype to fonts? like `/usr/local/share/fonts/newfonts` and `/usr/share/fonts/newfonts`. Revert me if it does not work, I will look at it with this font.

Hi agiratech-reddysai
